woocommerce/woocommerce
# [Product Block Editor] Remove additional create term modal

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR removes an extra step in the process of creating attribute terms. Previously, a modal was used to create new terms. With this update, the term can now be directly created by clicking on `+ Create "[term-name]"`. 

Closes #39223.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a global attribute and a few terms under Products > Attributes. 
2. Enable the product blocks editor under WooCommerce -> Settings -> Advanced -> Features.
3. Navigate to the add product page.
4. Click on `Organization` and then on the `Add new` button located under `Attributes`.
5. Choose the attribute you have already set up. The values you've created should appear in the dropdown menu.
6. Write a new term name in the input under the `Values` column and press `+ Create "[term-name]"`.
7. Confirm that the new term has been appended to the list of terms in the input field.
8. Verify that all functions perform as expected.

![Screen Capture on 2023-08-08 at 12-34-18](https://github.com/woocommerce/woocommerce/assets/1314156/1e867cbf-3515-4be3-9718-2c05aeb03747)



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
